endpoints.json

```diff
@@ -23,5 +23,77 @@
         }
       ]
     }
+  },
+
+  "GET /api/articles/:article_id": {
+    "description": "serves information about a specific article",
+    "queries": ["article_id"],
```

Comment:
Article_id isnt a query, its a parameter. 

You queries would be sort_by, order etc.